sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -44,6 +44,14 @@ public partial struct StorageTransferOptions : System.IEquatable<Azure.Storage.S
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public static bool operator !=(Azure.Storage.StorageTransferOptions left, Azure.Storage.StorageTransferOptions right) { throw null; }
     }
+    public partial class StorageTransferStatus
+    {
+        public StorageTransferStatus() { }
+        public long bytesTransferred { get { throw null; } }
+        public long failedTransfer { get { throw null; } }
+        public long skippedTransfer { get { throw null; } }
+        public long successfulTransfers { get { throw null; } }
```

Comment:
should these be properties? btw. 2nd and 3rd is singular but 4th is plural.

these will keep accumulating aggregate counters, should we have a place where user can subscribe event listener to listen to each transfer status?

Reply:
Yeah they're properties. I think my aim was to making something like
IProgress<Response<StorageTransferStatus>> ProgressHandler. I will try to add this in my next iteration so it's more clear.

Reply:
@rnpmsft is going to work on this when he converts this to the BlobDirectoryUploadResponse. It will be in a separate PR.